src/main/groovy/wooga/gradle/unity/UnityPluginExtension.groovy

```diff
@@ -84,6 +90,20 @@ trait UnityPluginExtension implements UnitySpec,
         assetsDir.set(value)
     }
 
+    private final DirectoryProperty packagesDir = objects.directoryProperty()
+
+    /**
+     * @return The Packages directory, where all resolved packages by an Unity project are recorded
+     */
+    @Internal
+    DirectoryProperty getPackagesDir() {
+        packagesDir
+    }
+
+    void setPackagesDir(Provider<Directory> value) {
+        packagesDir.set(value)
+    }
```

Comment:
To get in line with most other getter/setter I would also add a `File` setter for `packagesDir`

Reply:
Sure.